### PR TITLE
Combustor model and Emission Index estimation

### PR DESCRIPTION
**Pull request by MGuidotti**
The Combustor model is developed using Cantera, based on a Chemical Reactor Network formulation. The Primary Zone of the Combustor is modeled using Perfectly Stirred Reactors in parallel, while the Secondary Zone and Dilution Zone are modeled utilizing Plug Flow Reactors and mixers for the addition of dilution air. Emission Indexes can then be retrieved from a direct simulation of the model or using a surrogate model.

### CODE REVIEW

File: RCAIDE/Framework/Analyses/Aerodynamics/Aerodynamics.py [hunk 1]
```diff
@@ -48,7 +48,7 @@ def __defaults__(self):
         N/A
         """           
         self.tag                               = 'aerodynamics'  
-        self.geometry                          = Data()
+        self.vehicle                          = Data()
```
Comment (mclarke2) [inline RCAIDE/Framework/Analyses/Aerodynamics/Aerodynamics.py:51]: Align equal signs

File: RCAIDE/Framework/Analyses/Emissions/Emission_Index_CRN_Method.py [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+        # build the evaluation process
+        compute                         = Process()  
+        compute.emissions               = None  
+        self.process                    = Process()
+        self.process.compute            = compute        
+        
+        return
+    
+    def initialize(self):  
+        use_surrogate   = self.settings.use_surrogate
```
Comment (mclarke2) [inline RCAIDE/Framework/Analyses/Emissions/Emission_Index_CRN_Method.py:54]: add function header

File: RCAIDE/Framework/Analyses/Emissions/Emission_Index_Correlation_Method.py [hunk 3]
```diff
@@ -27,26 +28,43 @@ def __defaults__(self):
...
+        # build the evaluation process
+        compute                         = Process()  
+        compute.emissions               = None  
+        self.process                    = Process()
+        self.process.compute            = compute 
+                
         return
             
+    def initialize(self):   
+        compute   =  self.process.compute
```
Comment (mclarke2) [inline RCAIDE/Framework/Analyses/Emissions/Emission_Index_Correlation_Method.py:42]: add functiom header

File: RCAIDE/Framework/Analyses/Stability/Stability.py [hunk 4]
```diff
@@ -37,8 +37,8 @@ def __defaults__(self):
         Properties Used:
         N/A
         """   
-        self.tag    = 'stability'
-        self.geometry = Data()
+        self.tag     = 'stability'
```
Comment (mclarke2) [inline RCAIDE/Framework/Analyses/Stability/Stability.py:40]: align equals signs

File: RCAIDE/Library/Methods/Aerodynamics/Vortex_Lattice_Method/evaluate_VLM.py [hunk 5]
```diff
@@ -31,7 +31,7 @@ def evaluate_surrogate(state,settings,geometry):
         aerodynamics : VLM analysis  [unitless]
         state        : flight conditions     [unitless]
         settings     : VLM analysis settings [unitless]
-        geometry     : vehicle configuration [unitless] 
+        vehicle     : vehicle configuration [unitless]
```
Comment (mclarke2) [inline RCAIDE/Library/Methods/Aerodynamics/Vortex_Lattice_Method/evaluate_VLM.py:34]: align

File: RCAIDE/Library/Methods/Propulsors/Turbofan_Propulsor/compute_thurst.py [hunk 6]
```diff
@@ -44,9 +44,9 @@ def compute_thrust(turbofan,turbofan_conditions,conditions):
            freestream.gravity                            (numpy.ndarray): freestream gravity            [m/s^2] 
            propulsion.throttle                           (numpy.ndarray): throttle                      [unitless] 
         turbofan 
-           ..fuel_to_air_ratio                          (float): fuel_to_air_ratio                    [unitless] 
-           ..total_temperature_reference                (float): total_temperature_reference          [K] 
-           ..total_pressure_reference                   (float): total_pressure_reference             [Pa]    
+           .fuel_to_air_ratio                          (float): fuel_to_air_ratio                    [unitless] 
+           .total_temperature_reference                (float): total_temperature_reference          [K] 
+           .total_pressure_reference                   (float): total_pressure_reference             [Pa]
```
Comment (mclarke2) [inline RCAIDE/Library/Methods/Propulsors/Turbofan_Propulsor/compute_thurst.py:49]: align units

File: RCAIDE/Library/Methods/Propulsors/Turbofan_Propulsor/compute_thurst.py [hunk 7]
```diff
@@ -116,7 +116,7 @@ def compute_thrust(turbofan,turbofan_conditions,conditions):
 
     # Pack turbofan outouts  
     turbofan_conditions.thrust                            = FD2 
-    turbofan_conditions.thrust_specific_fuel_consumption  = TSFC
+    turbofan_conditions.thrust_specific_fuel_consumption  = TSFC/ Units.hour
```
Comment (mclarke2) [inline RCAIDE/Library/Methods/Propulsors/Turbofan_Propulsor/compute_thurst.py:119]: Make sure this is in the right units

**Review by mclarke2 — COMMENTED** [review]
Small fixed

File: RCAIDE/Framework/Analyses/Emissions/Emission_Index_CRN_Method.py (hunk 2 above)
Comment (MGuidotti) [reply RCAIDE/Framework/Analyses/Emissions/Emission_Index_CRN_Method.py:54]: Done, thanks!

File: RCAIDE/Framework/Analyses/Emissions/Emission_Index_Correlation_Method.py (hunk 3 above)
Comment (MGuidotti) [reply RCAIDE/Framework/Analyses/Emissions/Emission_Index_Correlation_Method.py:42]: Done

File: RCAIDE/Framework/Analyses/Stability/Stability.py (hunk 4 above)
Comment (MGuidotti) [reply RCAIDE/Framework/Analyses/Stability/Stability.py:40]: Completed

File: RCAIDE/Library/Methods/Aerodynamics/Vortex_Lattice_Method/evaluate_VLM.py (hunk 5 above)
Comment (MGuidotti) [reply RCAIDE/Library/Methods/Aerodynamics/Vortex_Lattice_Method/evaluate_VLM.py:34]: Done

File: RCAIDE/Library/Methods/Propulsors/Turbofan_Propulsor/compute_thurst.py (hunk 6 above)
Comment (MGuidotti) [reply RCAIDE/Library/Methods/Propulsors/Turbofan_Propulsor/compute_thurst.py:49]: Done, thanks!

File: RCAIDE/Library/Methods/Propulsors/Turbofan_Propulsor/compute_thurst.py (hunk 7 above)
Comment (MGuidotti) [reply RCAIDE/Library/Methods/Propulsors/Turbofan_Propulsor/compute_thurst.py]: Fixed it, thanks!

**Review by mclarke2 — APPROVED** [review]

**Pull request merged**